Source/Manager/Identifiers/PhoneNumber.swift

```diff
@@ -85,35 +79,42 @@ public struct PhoneNumber: IdentifierProtocol {
      */
     public init?(fullNumber: String?) {
         guard let fullNumber = fullNumber else { return nil }
-        guard let normalizedPhoneNumber = PhoneNumber.normalize(fullNumber, isCountryCode: true) else { return nil }
+        guard let normalizedPhoneNumber = PhoneNumber.normalizeFullNumber(fullNumber) else { return nil }
         self.phoneNumber = fullNumber
         self.normalizedPhoneNumber = normalizedPhoneNumber
         self.numberOfDigitsInNormalizedCountryCode = nil
     }
 
-    static func normalize(_ identifier: String, isCountryCode: Bool) -> String? {
-        guard identifier.rangeOfCharacter(from: CharacterSet.phoneNumberCharacterSet().inverted) == nil else {
-            return nil
-        }
+    private static func normalizeFullNumber(_ value: String) -> String? {
+        var string = value.filter { $0 != " " }
 
-        let decimalDigitCharacters = identifier.filter {
-            String($0).rangeOfCharacter(from: CharacterSet.decimalDigits) != nil
+        if string.hasPrefix("+") {
+            string.remove(at: string.startIndex)
+        } else if string.hasPrefix("00") {
+            string.removeFirst(2)
+        } else {
+            return nil
         }
 
-        guard decimalDigitCharacters.count > 0 else {
+        guard string.count > 0 && string.rangeOfCharacter(from: CharacterSet.decimalDigits.inverted) == nil else {
```

Comment:
Do you need the `string.count > 0`? Looks like it gives the same result without it.

Reply:
Si! incase only + or 00 is passed in